Ginger/GingerCoreNET/Run/RunSetConfig.cs

```diff
@@ -86,8 +88,36 @@ public string Description
             }
         }
 
+        private ObservableList<GingerRunner> mGingerRunners;
+        /// <summary>
+        /// Been used to identify if Activity Variables were lazy loaded already or not
+        /// </summary>
+        public bool GingerRunnersLazyLoad { get { return (mGingerRunners != null) ? mGingerRunners.LazyLoad : false; } }
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5970337)